src/Api/Tools/Validators/SendRotationValidator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Bit.Api.Auth.Validators;
+using Bit.Api.Tools.Models.Request;
+using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Tools.Entities;
+using Bit.Core.Tools.Repositories;
+using Bit.Core.Tools.Services;
+
+namespace Bit.Api.Tools.Validators;
+
+public class SendRotationValidator : IRotationValidator<IEnumerable<SendWithIdRequestModel>, IEnumerable<Send>>
+{
+    private readonly ISendService _sendService;
+    private readonly ISendRepository _sendRepository;
+
+    public SendRotationValidator(ISendService sendService, ISendRepository sendRepository)
+    {
+        _sendService = sendService;
+        _sendRepository = sendRepository;
+    }
+
+    public async Task<IEnumerable<Send>> ValidateAsync(User user, IEnumerable<SendWithIdRequestModel> sends)
```

Comment:
Added!

Reply:
Should have specified that the `ValidateAsync` inherits the documentation from the interface. Adding documentation for each of these feels overkill since its purpose is basically the same.

Not sure I see the benefit of adding documentation to a constructor in this situation, since DI handles the instantiation.

Reply:
Documentation added to constructor